Code/GraphMol/Wrap/testSGroups.py

```diff
@@ -98,6 +98,31 @@ def testLifetime(self):
         parent = sgl[0].GetOwningMol()
         self.assertEqual(smi, Chem.MolToSmiles(parent))
 
+    def testHRemoval(self):
```

Comment:
Having an equivalent test in C++ would also be good.

Reply:
Makes sense. I was just lazy to write the C++ test...